### ISSUE TITLE
spideroak is out of date and inoperable

### ISSUE THREAD

**Issue by goertzenator** [NixOS/nixpkgs#36862]
## Issue description

spideroak-6.1.9 is out of date and inoperable due to the expired SSL key present in this version.

Current version is 7.0.1 released 1 Feb 2018.

### Steps to reproduce
1. Run spideroak-6.1.9
2. Login with valid credentials.
3. Session is rejected due to [expired SSL key](https://support.spideroak.com/hc/en-us/articles/115005874603-SSL-Certificate-Verification-Error).

## Technical details
```
# nix-shell -p nix-info --run "nix-info -m"
(snip)
 - system: `"x86_64-linux"`
 - host os: `Linux 4.9.86, NixOS, 17.09.3146.084445b8f38 (Hummingbird)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
 - channels(root): `"nixos-17.09.3146.084445b8f38"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`
```

Notifying @neilmayhew since github says he touched it last.

**Comment by jtojnar** [comment]
https://github.com/NixOS/nixpkgs/pull/36844

**Comment by fpletz** [comment]
Fixed by #36844.
